sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/generators.scala

```diff
@@ -300,7 +311,7 @@ abstract class ExplodeBase extends UnaryExpression with CollectionGenerator with
 case class Explode(child: Expression) extends ExplodeBase {
   override val position: Boolean = false
 }
-
+class OuterExplode(child: Expression) extends GeneratorOuter(Explode(child))
```

Comment:
I made it like this so I can use it in FunctionRegistry.scala: expression[OuterExplode]("outer_explode")
Is there a way I can still make it work with GeneratorOuter(Explode(child)) ?